pkg/reconciler/openshift/openshift.go

```diff
@@ -147,6 +147,16 @@ func policyRulesForClusterConfig() []rbacv1.PolicyRule {
 			Verbs: []string{
 				"*",
 			},
+		}, {
+			APIGroups: []string{
+				"compliance.openshift.io",
+			},
+			Resources: []string{
+				"*",
+			},
+			Verbs: []string{
+				"*",
+			},
```

Comment:
Looks like we are openning too much.  Letting in the entire api group is not needed per https://jaosorior.dev/2021/compliance-operator-gitops/

```
rules:
  - verbs:
      - get
      - watch
      - list
    apiGroups:
      - compliance.openshift.io
    resources:
      - scansettings
      - compliancesuites
      - compliancescans
      - compliancecheckresults
      - complianceremediations
  - verbs:
      - '*'
    apiGroups:
      - compliance.openshift.io
    resources:
      - scansettingbindings
      ```

Reply:
Thanks William, done!!